src/intersection-observer-polyfill.js

```diff
@@ -38,6 +38,8 @@ export const DEFAULT_THRESHOLD =
     [0, 0.05, 0.1, 0.15, 0.2, 0.25, 0.3, 0.35, 0.4,
     0.45, 0.5, 0.55, 0.6, 0.65, 0.7, 0.75, 0.8, 0.85, 0.9, 0.95, 1];
 
+const INIT_TIME = Date.now();
```

Comment:
presubmit is passing locally for me. 
Well if I can't do this where should I get the init time? I can create one in `IntersectionObserverPolyfill`, but what about the export function `#getIntersectionChangeEntry`?